java/src/main/java/ai/onnxruntime/OrtSession.java

```diff
@@ -1273,6 +1272,37 @@ public void addCoreML(EnumSet<CoreMLFlags> flags) throws OrtException {
       addCoreML(OnnxRuntime.ortApiHandle, nativeHandle, OrtFlags.aggregateToInt(flags));
     }
 
+    /**
+     * Adds the specified execution provider and device tuples as an execution backend.
```

Comment:
Would it be worthwhile to mention that the order EPs are added in is the priority order for node assignment? First one is the highest priority.

This is the same as for the existing AddExecutionProvider so maybe not required as we've gotten this far without that info. :)

Reply:
I've added a note to the top of this class explaining EP assignment, and updated this method's javadoc.